src/Symfony/Component/Form/Form.php

```diff
@@ -1050,7 +1050,13 @@ private function modelToNorm($value)
                 $value = $transformer->transform($value);
             }
         } catch (TransformationFailedException $exception) {
-            throw new TransformationFailedException('Unable to transform value for property path "'.$this->getPropertyPath().'": '.$exception->getMessage(), $exception->getCode(), $exception);
+            throw new TransformationFailedException(
```

Comment:
should be kept on one line

Reply:
also, not sure if we really need this change

Reply:
> not sure if we really need this change

Well, we need to forward the invalid message and parameters to the new exception, otherwise it's just vanished. What did you mean?